docs/timeline/index.html

```diff
@@ -1571,7 +1577,7 @@ <h2 id="Editing_Items">Editing Items</h2>
 };</pre>
```

Comment:
Here in this code example the new option `overrideItems` is missing. Should be added.